src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3874,18 +3874,18 @@
      * </pre>
      *
      * <p>parses as:</p>
-     * <pre>
-     * +--LITERAL_NEW (new)
-     *     |
-     *     +--IDENT (ArrayList)
-     *     +--LPAREN (()
-     *     +--ELIST
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--NUM_INT (50)
-     *     +--RPAREN ())
-     * </pre>
+     *       <pre>
```

Comment:
Remove indentation before `<pre>`(the extra part is appearing in dark green before `<pre>`), also remove similar from `</pre>` below.